ginivision/src/main/java/net/gini/android/vision/camera/CameraFragmentImpl.java

```diff
@@ -428,7 +428,8 @@ boolean onActivityResult(final int requestCode, final int resultCode, final Inte
                     }
                     Document document = DocumentFactory.newDocumentFromIntent(data, activity,
                             DeviceHelper.getDeviceOrientation(activity),
-                            DeviceHelper.getDeviceType(activity));
+                            DeviceHelper.getDeviceType(activity),
+                            "picker");
```

Comment:
maybe an enum could make sense here, or some other constants, that would make it easier to edit/add later.

Reply:
I agree, I created #95 for this to avoid wider changes for now.